Jenkinsfiles/Jenkinsfile-stratum-bf-build.groovy

```diff
@@ -14,28 +13,31 @@ pipeline {
     }
     options {
         timeout(time: 60, unit: 'MINUTES')
+        withAWS(credentials:'AKIAR6Z4ESHHIXKSMOXU')
```

Comment:
Should we use a credentials variable here, instead of hardcoding this in? (adding a credential itself must be done through Jenkins UI)

Reply:
Agree, already added the credential through Jenkins UI. I can add this as a variable in defaults.yaml and reference it.